### PR TITLE
migrated to null-safety, fixed deprecation errors

### PR DESCRIPTION
**Pull request by jabguru**
Thanks for the package, using it but needed to migrate to null safety.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,11 +1,10 @@
 name: humanitarian_icons
 description: Humanitarian Icons package.
 version: 0.0.1
-author: Dhruvil Patel
```
Comment (dhruvilp) [inline pubspec.yaml:4]: replace with 
```maintainer: Dhruvil Patel (@dhruvilp )```

**Review by dhruvilp — CHANGES_REQUESTED** [review]
Hey,

Thanks for raising a PR, really appreciate it!
Can you please ignore/delete these files/folders and commit again:

.dart_tool/package_config.json
.dart_tool/package_config_subset
.dart_tool/version
.packages
example/.dart_tool/package_config.json
example/.dart_tool/package_config_subset
example/.dart_tool/version
example/.packages
example/pubspec.lock
pubspec.lock

It's not good practice to commit files and folders that start with ".".

Thanks man!

@jabguru

**Comment by jabguru** [comment]
Alright, made the changes now, you can check it out, thanks.

**Review by dhruvilp — APPROVED** [review]
Perfect, thanks!

**Pull request merged**